### DISCUSSION TITLE
switch-preset-column-width but backwards

### DISCUSSION THREAD

**Discussion by arm32x** [YaLTeR/niri#836] [Ideas]
As far as I can tell, the `switch-preset-column-width` and `switch-preset-window-height` actions can only cycle forwards through presets. I have a decent number of presets configured, so it would be nice to have a second keybind to cycle backwards through them to access the one I want quicker.

**Comment by YaLTeR** [comment]
I guess some kind of switch-backward-preset-column-width (name suggestions welcome) would be alright and relatively straightforward.
